app/assets/stylesheets/components/_buttons.scss

```diff
@@ -80,6 +80,49 @@
   }
 }
 
+.button--green {
```

Comment:
I thought that `button--green` can be re-used in the future if we want more money green buttons. And that `button--intercom` below has very specific styles for the intercom button.